lib/streamlit/runtime/scriptrunner/magic.py

```diff
@@ -119,8 +137,6 @@ def _modify_ast_subtree(
 
     ast.fix_missing_locations(tree)
 
-    return tree
```

Comment:
`_modify_ast_subtree` is used in many places expecting it changes the passed object in-place, so I removed the returned value as it's rather confusing.
There were some lines using the returned value, so I modified them as well.